Makefile

```diff
@@ -13,3 +13,9 @@ clean:
 
 check: all
 	@scripts/verify.sh
+
+CC := gcc
+BLOCKLIST := blocklist_user
+
+blocklist:
```

Comment:
Don't do this. Check https://github.com/sysprog21/vcam/blob/master/Makefile for consistent build rules.

Reply:
@jserv 
I've made some adjustment according to the suggestion, though I'm not sure if I wrote it in the form that is desired or not. Please let me know if I need to make any correction, thank you.